wrapanapi/systems/redfish.py

```diff
@@ -168,6 +225,82 @@ def server_inventory(self, physical_server, requested_items, **kwargs):
         return {item: self._server_inventory_available[item](redfish_server)
                 for item in requested_items}
 
+    def rack_stats(self, physical_rack, requested_stats, **kwargs):
```

Comment:
Do we plan to use the `kwargs` or should we just omit it from the list?

Reply:
👍 removed.

Reply:
`**kwargs` is still in the method definition, I'm starting to think that a commit from @matejart has not been pushed to this fork/PR?

Reply:
This is embarrassing. I in fact forgot to push these changes. The `**kwargs` are now gone.